make-money/monetization-network.md

```diff
@@ -0,0 +1,38 @@
+# Monetization
+
+Tap for Tap has combined some big time advertisers, technologies and other developers on the Tap for Tap network to bring you an inventory of thousands of quality advertisements you can display in your app for money.
+
+In order to earn revenue from our monetization network you will need to:
+
+1. Download our [SDK](doc/getting-started)
+2. Have your [Revenue Preference Slider](doc/how-it-works) set to Make Money > 0%
+3. Place [ad units](doc/make-money/ad-placement-overview) in your application
+
+## Ad Inventory
+
+To find out the types of ads we support check out our [ad overview guide](doc/make-money/ad-overview). You can also take [steps](/doc/make-money/dos-donts) to improve the performance of your ads by ensuring proper placement and visibility.
+
+
+## Performance
+
+One of the main factors that determines how well your ads perform is where your audience is located. Tap for Tap has grouped each country into [tiers](/doc/country-tiers)  based on perceived performace. Tier 1 countries generally make a lot more revenue than tier 4. 
+
+## 100% Fill Rate
+
+A unique feature of Tap for Tap is we will always fall back on a [tap exchange](/doc/tap-exchange) ad if we cannot find a good revenue generating ad to place.  This allows you to always benefit from placing a Tap for Tap ad.
+
+## Demographics
+
+Providing user information will increase your revenue potential as it allows for better targeting. We accept four pieces of information about users: gender, year of birth, location and account id (on your system).  Providing this information both protects your users identity and improves revenue performance for your traffic.
+
+## Optimization
+
+Over time Tap for Tap rotates numerous apps and ads on your monetization traffic to find the best performing solution.  Overtime you can expect greater performance.
+
+## Fraudulent Traffic
+
```

Comment:
The language on the second sentence is weird... 'We have seen everything from generating fake impression..." is kinda confrontational? I dunno. Should probably keep a more neutral tone. Something like:

"Tap for Tap takes protecting its developers and advertisers seriously. We have a zero tolerance policy for fraud. Fraudulent actions include generating fake impressions, generating fake taps, and installing apps from ads to increase revenue. If you are found to be generating fraudulent traffic, payments will be withheld and legal action may be pursued."